content/geometry/object-environment/environment-structure.adoc

```diff
@@ -0,0 +1,433 @@
+////
+
+Add a figure.
+
+.<Figure caption>
+//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
+
+== Grp_Drivable_Area
+
+The drivable area includes any surfaces that are part of the road structure including lanes and parking areas. It is comparable to the scope of the OpenDrive standard for on-road use cases.
+
+////
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+////
+
+Add a figure.
+
+.<Figure caption>
+//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
+
+== Grp_Sidewalks 
+
+Sidewalks are traffic spaces that are dedicated for pedestrians and sometimes cyclists. 
+
+////
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+////
+
+Add a figure.
+
+.<Figure caption>
+
+
+
+
+== Grp_Road_Marks
+
+
+Road marks include any markings on the road and traffic spaces.
+
+////
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+////
+
+Add a figure.
+
+.<Figure caption>
+//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
+
+== Grp_Road_Objects
+
+Road Objects include all other objects on or nearby the road, excluding signals, i.e., signs and traffic lights.
+////
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+////
+
+Add a figure.
+
+.<Figure caption>
+
+
+== Grp_Signals
+
+
+Traffic signals, including signs and lights. Signals describe the relevant area or volume of a traffic signal only. Posts and gantrys are considered road objects.
+
+////
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+
+////
+Add a figure.
+
+.<Figure caption>
+
+
+
+== Grp_Sign_<signal_idx> (T)
+
+
+Traffic signs belong to traffic signals. Therefor, they are indexed using a signal-idex and cannot have the same ID as a traffic light, which is also enumerated by the signal index. The signal index can be taken over from OpenDRIVE or OSI by iterating over all signals from (0,...,n).
+
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+
+
+Add a figure.
+
+.<Figure caption>
+
+
+== Grp_Traffic_Light_<signal_idx> (T)
+
+
+Traffic lights belong to traffic signals to provide temporary changes in illumination. The signal index can be taken over from OpenDRIVE or OSI by iterating over all signals from (0,...,n). The indices are used for both traffic signs and traffic lights. A  traffic light cannot have the same index ID as a traffic sign in a single environment. 
```

Comment:
The signal index can be taken over from OpenDRIVE or OSI **or (if none of the latter exists) it can be generated** by [...]